build.sbt

```diff
@@ -94,8 +94,10 @@ lazy val core = crossProject(JSPlatform, JVMPlatform)
     libraryDependencies ++= Seq(
       "org.typelevel" %%% "cats-core" % catsVersion,
       "org.typelevel" %%% "cats-effect-kernel" % catsEffectVersion,
+      "org.typelevel" %%% "cats-effect-std" % catsEffectVersion,
```

Comment:
Needed for `Random.javaSecuritySecureRandom`

Reply:
See also:
- https://github.com/typelevel/bobcats/issues/82